tests/performance/collect_measurements

```diff
@@ -68,10 +69,11 @@ done | sort | tr $'\t' ',')"
 #
 
 (
-  echo "# Date:     $(date -u --rfc-3339=s)"
-  echo "# Site:     ${site}"
-  echo "# Branch:   $branch"
-  echo "# Revision: $revision"
+  echo "# Date:      $(date -u --rfc-3339=s)"
+  echo "# Site:      ${site}"
+  echo "# Branch:    ${branch}"
+  echo "# Revision:  ${revision}"
+  echo "# Timestamp: ${timestamp}"
```

Comment:
Out of curiosity, is this always going to be the timestamp of a merge commit? I'm asking because these are time ordered, whereas the individual commits we merge are of course not.

Reply:
This is going to be a merge commit most of the time - at least when you simply check out the master branch.